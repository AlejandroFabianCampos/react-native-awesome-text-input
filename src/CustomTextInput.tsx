import React from 'react'
import { TextInput } from 'react-native'

export const CustomTextInput = ({...props}) => {
    return (
        <TextInput {...props}/>
    )
}
