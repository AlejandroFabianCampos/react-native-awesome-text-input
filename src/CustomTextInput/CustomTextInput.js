import React from 'react'
import { TextInput, View } from 'react-native'

export const CustomTextInput = ({...props}) => (
    <View>
        <TextInput {...props}/>
    </View>
)
