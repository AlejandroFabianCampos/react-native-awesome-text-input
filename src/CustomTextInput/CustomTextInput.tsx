import * as React from 'react'
import { TextInput, View, TextInputProps } from 'react-native'

export const CustomTextInput: React.FC<TextInputProps> = ({...props}) => (
    <View>
        <TextInput {...props}/>
    </View>
)
