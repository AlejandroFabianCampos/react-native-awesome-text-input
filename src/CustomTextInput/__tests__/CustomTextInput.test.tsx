import React from 'react';
import { CustomTextInput } from '../CustomTextInput';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<CustomTextInput placeholder="Placeholder" />).toJSON();
  expect(tree).toMatchSnapshot();
});