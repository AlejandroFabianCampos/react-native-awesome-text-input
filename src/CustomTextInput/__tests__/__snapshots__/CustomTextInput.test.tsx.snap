// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<TextInput
  allowFontScaling={true}
  placeholder="Placeholder"
  rejectResponderTermination={true}
  underlineColorAndroid="transparent"
/>
`;
